### ISSUE TITLE
"HDMI-Settings > Content type" doesn't do anything on my PC and "unchanged" option causes error message

### ISSUE THREAD

**Issue by macadamiaz** [Maassoft/ColorControl#439]
On an Nvidia RTX 4070 Ti SUPER, latest drivers 566.03, Windows 10:

Changing "HDMI-Settings > Content type" doesn't do anything on my PC (It works to change it in the nvidia control panel)
I tried all options, Game, Cinema etc and made sure HDMI-Settings is included.
Additionally, i get the following error Popup when i choose "unchanged":

![hdmi content type unchanged](https://github.com/user-attachments/assets/ad3bb57e-3486-431d-9f27-8264993e7057)

**Comment by Maassoft** [comment]
If you change it in ColorControl the "Content type reported to the display" isn't changed in the Nvidia Control Panel?
What display do you use? If I set the "Game mode" on my Samsung tv to "Auto" it switches automatically whenever I change the content type in ColorControl from Game to Graphics or back again.

**Comment by macadamiaz** [comment]
Exactly, it isn't changed in the nvidia control panel. I also use a Samsung TV, S90C on Game mode Auto, and it doesn't switch through colorcontrol, but does when i change the setting in the nvidia control panel.

The options are also a bit different than what's available in colorcontrol: in nvidia i have Auto, Desktop apps, Games and Fullscreen video (translated, not sure about the exact wording in english) but even switching between auto and game mode, which are available in nvidia panel and colorcontrol doesn't work.
